README.md

```diff
@@ -264,9 +267,8 @@ To deploy the app to the cluster, run the following command:
 For example:
 
 ```
-SUBMISSION_BUCKET_NAME=census-eq-dev-1234567-survey-runner-submission ./k8s/deploy_app.sh
+SUBMISSION_BUCKET_NAME=project-id-survey-runner-submission ./k8s/deploy_app.sh
```

Comment:
This no longer works since the defaults were removed from the script.

Reply:
Changed the README slightly - now states that env vars *should* be set but that some are optional.